static/json/languages/en_US.json

```diff
@@ -1458,14 +1458,18 @@
     "approvals": "Token Approvals",
     "this_token_is_on_network": "This %{symbol} is on the %{chainName} network",
     "view_on": "View on %{explorer}",
+    "view_on_coingecko": "View on CoinGecko",
     "not_available": "Price not available",
     "more_options": {
       "copy_address": "Copy Address",
       "view_on_explorer": "View on %{explorer}",
       "hide": "Hide Token",
       "report": "Report Token",
       "unpin_token": "Unpin %{name}",
-      "pin_token": "Pin %{name}"
+      "pin_token": "Pin %{name}",
+      "hide_token": "Hide %{name}",
+      "unhide_token": "Unhide %{name}",
+      "hide_token_toast": "%{name} hidden"
```

Comment:
Check out the `hide_balances_toast` example; I think we want to bump this out of this section and have a separate toast key for these actions. Try this for a string: `Your %{name} is hidden`. Would also need a toast for unhide and pin/unpin I think. I realized after my prior comment that we do want a toast on the Token Details page for those, but don't need one on Tokens (because of the visual list shift)